android/src/main/java/com/bitmovin/player/reactnative/converter/JsonConverter.kt

```diff
@@ -1169,6 +1170,59 @@ class JsonConverter {
             playerViewConfig = toPlayerViewConfig(json),
             pictureInPictureConfig = toPictureInPictureConfig(json.getMap("pictureInPictureConfig")),
         )
+
+        @JvmStatic
+        fun fromBufferLevel(bufferLevel: BufferLevel?): WritableMap? {
+            if (bufferLevel == null) {
+                return null
+            }
+
+            return Arguments.createMap().apply {
+                putDouble("level", bufferLevel.level)
+                putDouble("targetLevel", bufferLevel.targetLevel)
+                putInt(
+                    "media",
+                    when (bufferLevel.media) {
+                        MediaType.Audio -> 0
+                        MediaType.Video -> 1
```

Comment:
I would extract this to a function and instead of integers I would use strings. That makes those values easier to debug when they are serialized. 
Simething like: `fromAdSourceType`

Reply:
Thanks for providing an example to look at!
Address in 5b8f4499f9f9f61b25aa2da976471ed5bfccbb67

Reply:
I added an additional comment here: https://github.com/bitmovin/bitmovin-player-react-native/commit/5b8f4499f9f9f61b25aa2da976471ed5bfccbb67#r130700746

Reply:
Address in 8f4340eb71a7db365b908159dadbb2736fe86692